### ISSUE TITLE
Add Discord link to README and website

### ISSUE THREAD

**Issue by ramedina86** [streamsync-cloud/streamsync#221]
- Add badge to README
- Add link/icon to docs website

**Comment by FabienArcellier** [comment]
Here is a code I use for one of my repo

![image](https://github.com/streamsync-cloud/streamsync/assets/159559/1db98375-2762-4caf-b193-90af58a19c60)

```markdown
[![PyPi](https://img.shields.io/pypi/v/alfred-cli.svg?label=Version)](https://pypi.org/project/alfred-cli/)
[![Python](https://img.shields.io/pypi/pyversions/alfred-cli.svg)](https://pypi.org/project/alfred-cli/)
[![CI](https://github.com/FabienArcellier/alfred-cli/actions/workflows/ci.yml/badge.svg)](https://github.com/FabienArcellier/alfred-cli/actions/workflows/ci.yml) [![CI-Windows](https://github.com/FabienArcellier/alfred-cli/actions/workflows/ci-windows.yml/badge.svg)](https://github.com/FabienArcellier/alfred-cli/actions/workflows/ci-windows.yml)
[![Documentation Status](https://readthedocs.org/projects/alfred-cli/badge/?version=latest)](https://alfred-cli.readthedocs.io/en/latest/?badge=latest)
[![Discord](https://img.shields.io/badge/discord-alfred-5865F2?logo=discord&logoColor=white)](https://discord.gg/nMn9YPRGSY)
[![License](https://img.shields.io/badge/license-MIT-007EC7.svg)](LICENSE)
```

**Comment by FabienArcellier** [comment]
done on the branch dev : https://github.com/streamsync-cloud/streamsync/blob/dev/README.md

**Comment by ramedina86** [comment]
Shipped in 0.4.0 update.
